esp32_ulp/preprocess.py

```diff
@@ -2,6 +2,34 @@
 from .util import split_tokens
 
 
+class RTC_Macros:
+    @staticmethod
+    def READ_RTC_REG(rtc_reg, low_bit, bit_width):
+        return '\treg_rd ' + ', '.join((
+            rtc_reg,
+            '%s + %s - 1' % (low_bit, bit_width),
+            low_bit
+        ))
+
+    @staticmethod
+    def WRITE_RTC_REG(rtc_reg, low_bit, bit_width, value):
+        args = (
+            rtc_reg,
+            '%s + %s - 1' % (low_bit, bit_width),
+            low_bit,
+            value
+        )
+        return '\treg_wr ' + ', '.join(args)
```

Comment:
maybe do it without `args` (see READ_... above).

Reply:
:) the READ_.. case above I tried out the different approach, but didn't mean to commit it. But i have changed them both that approach now.